x-pack/metricbeat/module/aws/_meta/docs.asciidoc

```diff
@@ -23,7 +23,7 @@ Please see <<aws-credentials-config,AWS credentials options>> for more details.
 This module also accepts optional configuration `regions` to specify which
 AWS regions to query metrics from. If the `regions` parameter is not set in the
 config file, then by default, the `aws` module will query metrics from all available
-AWS regions.
+AWS regions. If `endpoint` is specified, `regions` becomes a required config parameter.
```

Comment:
I'd suggest to add a section about connecting to AWS partitions, with an example of the settings needed to connect with one of them using `endpoint` and `region`.

Reply:
Good point! I will add a section for AWS partitions! Thank you for the review!